tests/integration/test_ffs.py

```diff
@@ -32,6 +32,40 @@ def test_grpc_ffs_add_to_hot(pygate_client: PowerGateClient, ffs_instance):
     assert res is not None
     assert res.cid is not None
 
+def test_grpc_ffs_add_then_get_content(pygate_client: PowerGateClient, ffs_instance):
+    res = pygate_client.ffs.add_to_hot(test_chunks(), ffs_instance.token)
+    
+    assert res is not None
+
+    pygate_client.ffs.push(res.cid, ffs_instance.token)
+    f = pygate_client.ffs.get(res.cid, ffs_instance.token)
+
+    assert next(f) == b'test_content'
+
+def test_grpc_ffs_list_wallet(pygate_client: PowerGateClient, ffs_instance):
+    res = pygate_client.ffs.addrs_list(token=ffs_instance.token)
+
+    assert res is not None
+    assert len(res.addrs) == 1
+    assert res.addrs[0].name == 'Initial Address'
+    assert res.addrs[0].type == 'bls'
+
+def test_create_new_wallet_address(pygate_client: PowerGateClient, ffs_instance: CreateResponse):
+    new_addr_name = 'test'
+    addr = pygate_client.ffs.addrs_new(name=new_addr_name, token=ffs_instance.token)
+
+    assert addr is not None
+    assert addr.addr is not None
+
+    res = pygate_client.ffs.addrs_list(token=ffs_instance.token)
+
+    assert res is not None
+    assert len(res.addrs) == 2
+    
+    expected_addr = AddrInfo(name=new_addr_name, addr=addr.addr, type='bls')
+    assert expected_addr in res.addrs
+    
+    
 def test_chunks():
     for _ in range(1):
-        yield AddToHotRequest(chunk=bytes("test_content", "ASCII"))
+        yield AddToHotRequest(chunk=bytes('test_content', 'ASCII'))
```

Comment:
Looking good man!! Just another minor thing, it seems that your IDE converts double quotes to single quotes, can you run `pipenv run format` before you push just to keep the quoting style consistent?

Reply:
This is a handy command! The formatting issue should be fixed now.